scripts/check-hardcoded-strings.sh

```
@@ -0,0 +1,104 @@
+# 4. label = { Text("...") } — catch simple one-liners that start with
+#    a letter inside lambda braces.
+scan 'label\s*=\s*\{\s*Text\("[[:alpha:]]' 'label'
+
+# 5. placeholder = "..." (e.g. TextField placeholder) with letter start.
+scan 'placeholder\s*=\s*"[[:alpha:]]' 'placeholder'
+
+if [[ $violations -gt 0 ]]; then
+    echo ""
+    echo "ERROR: $violations hardcoded user-facing string(s) found in:"
+    for f in "${RESOLVED[@]}"; do
+        echo "  - ${f#$ROOT/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)